romana/client/client.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
will package adaptor or package platform sounds good? package enviornment_adaptor will be bit too verbose?